docs/examples/agent/agent_runner/query_pipeline_agent.ipynb

```diff
@@ -52,7 +52,7 @@
   },
```

Comment:
<p>it seems it is not showing the graph tho</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/10563/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
good point, need to clear it